.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
+    types: [opened, synchronize, reopened]
+
+env:
+  DOCKER_BUILDKIT: 1
```

Comment:
You're not building any Docker images here, is this needed?

Reply:
aaaargh, you're right, not needed :see_no_evil: